<?xml version="1.0"?>
<launch>
  <arg name="map_path" default="$(find points_map_loader)/../../../data/pcd/room_scan1.pcd" />
  <arg name="map_frame" default="map" />
  <arg name="points_map_topic" default="points_map" />

  <node name="points_map_loader_node" pkg="points_map_loader" type="points_map_loader_node" output="screen" respawn="true" >
    <param name="map_path" value="$(arg map_path)" />
    <param name="map_frame" value="$(arg map_frame)" />
    <param name="points_map_topic" value="$(arg points_map_topic)" />
  </node>
</launch>
